src/common/legacy_config_opts.h

```diff
@@ -1659,6 +1659,7 @@ OPTION(rgw_log_http_headers, OPT_STR, "" ) // list of HTTP headers to log when s
 
 OPTION(rgw_num_async_rados_threads, OPT_INT, 32) // num of threads to use for async rados operations
 OPTION(rgw_md_notify_interval_msec, OPT_INT, 200) // metadata changes notification interval to followers
+OPTION(rgw_data_notify_interval_msec, OPT_INT, 200) // data changes notification interval to followers
```

Comment:
could you please add this to the new `common/options.cc` also? see the new entry for [rgw_md_notify_interval_msec](https://github.com/fangyuxiangGL/ceph/blob/ac2da9983605cd4506dfc26770da97fe398cb972/src/common/options.cc#L5268-L5270)